api/src/main/java/org/pmiops/workbench/db/dao/ParticipantCohortStatusDao.java

```diff
@@ -2,13 +2,32 @@
 
 import org.pmiops.workbench.db.model.ParticipantCohortStatus;
 import org.springframework.data.domain.Pageable;
-import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Slice;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.query.Param;
 
 public interface ParticipantCohortStatusDao extends CrudRepository<ParticipantCohortStatus, Long> {
 
+  // Important: Keep in sync with all DB rows that should be copied.
```

Comment:
@freemabd1 @danqrodney PTAL, this changed. Let me know if you have any other ideas on how to keep these two things in sync. This is a place where the ORM layer is useful.

Reply:
To expand slightly, the reason I can't just do a `*` is because I need to overwrite the `cohort_review_id` on clone. A temp table would be one solution in general, but that doesn't seem like a very good idea in this context.

Reply:
Yeah, MySQL doesn't give you any nice options here that I know of. (I love "EXCEPT foo" in Google SQL.)

You might be able to generate the list of columns via reflection on the Column annotations on ParticipantCohortStatus... but I'm not sure it's worth the trouble.